README.md

```diff
@@ -87,6 +87,9 @@ Currently, the following steps must be made to release a new version of tota11y:
    This requires someone with appropriate privileges.
 1. Run `npm publish`.
    This step will run tests and pre-publish checks, then perform a production build and publish the new package to NPM.
+   
+## Examples of Integration
+1. [webpack-react-typescript-project](https://github.com/azemetre/tota11y-webpack-react-typescript-example)
```

Comment:
Could you add some additional text before the list of links to preface them? Also, perhaps this should be a bulleted list rather than a numbered list? I think it will look more intentional that way, especially while there is only one item. It also won't imply any level of importance to any one entry.

Something along the lines of:

> Want to integrate tota11y into your site, but don't know where to start? Here are some examples from the tota11y community to inspire you.

Reply:
That you for the copy suggestion! I added it to the documentation and changed the section name from `Examples of Integration` to `Community Examples`, it reads better to me.What do you think?

I clarified what each linked example shows, hopefully it doesn't read a redundant. I got the idea from how Cypress links examples in their documentation on github:

https://github.com/cypress-io/code-coverage#examples